test/basic-querying.test.js

```diff
@@ -153,6 +153,18 @@ describe('basic-querying', function() {
           done();
         });
     });
+    it('should query by ids and null condition', function(done) {
+      User.findByIds([
+        createdUsers[0].id,
+        createdUsers[1].id],
+        {where: {vip: null}}, function(err, users) {
+          should.exist(users);
+          should.not.exist(err);
```

Comment:
@paulussup Could you please swap L161 with L162? It is always a good idea to check `err` first.